package.json

```diff
@@ -1,20 +1,19 @@
 {
   "name": "if-this-then-ad",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
This change constitutes a pretty big behavioral change. Not sure a patch-version bump makes this clear.